crates/forge-runner/src/lib.rs

```diff
@@ -245,21 +246,19 @@ impl TestCaseSummary<Fuzzing> {
                 test_statistics: (),
             } => {
                 let runs = results.len();
-                let gas_usages_vec = results
+                let gas_usages_vec: Vec<u128> = results
```

Comment:
I know it's out of scope, but why is this method implemented here and not in `test_case_summary.rs` 🤔